### DISCUSSION TITLE
how to not spawn webs on target

### DISCUSSION THREAD

**Discussion by Misaka-Mikoto-G** [winstxnhdw/lc-hax#242] [General]
I cannot inject if i delete the part of spider code in IEnemyPrompter.cs

**Comment by Quesoteric** [comment]
Are you sure you're deleting it correctly? [Refer to this](https://github.com/winstxnhdw/lc-hax/commit/824c523d70eaee244b9fdc962e0844b1c63d0c12)

**Reply by winstxnhdw** [reply]
You can just comment out the spawn web function

**Reply by Misaka-Mikoto-G** [reply]
void HandleBunkerSpider(SandSpiderAI bunkerSpider, PlayerControllerB targetPlayer, bool willTeleportEnemy) {
        /*this.TeleportEnemyToPlayer(bunkerSpider, targetPlayer, willTeleportEnemy, allowedInside: true);
        this.SetBehaviourState(bunkerSpider, BehaviourState.AGGRAVATED);

        Vector3 playerPosition = targetPlayer.transform.position;

        bunkerSpider.SpawnWebTrapServerRpc(
            playerPosition,
            playerPosition + (targetPlayer.transform.forward * 3.0f)
        );

        _ = bunkerSpider.Reflect()
                        .SetInternalField("watchFromDistance", false)?
                        .SetInternalField("chaseTimer", float.MaxValue);*/
    }

is this correct?

**Reply by winstxnhdw** [reply]
No, like this.

```cs
void HandleBunkerSpider(SandSpiderAI bunkerSpider, PlayerControllerB targetPlayer, bool willTeleportEnemy) {
    this.TeleportEnemyToPlayer(bunkerSpider, targetPlayer, willTeleportEnemy, allowedInside: true);
    this.SetBehaviourState(bunkerSpider, BehaviourState.AGGRAVATED);

    Vector3 playerPosition = targetPlayer.transform.position;

    /*bunkerSpider.SpawnWebTrapServerRpc(
        playerPosition,
        playerPosition + (targetPlayer.transform.forward * 3.0f)
    );*/

    _ = bunkerSpider.Reflect()
                    .SetInternalField("watchFromDistance", false)?
                    .SetInternalField("chaseTimer", float.MaxValue);*/
}
```

**Reply by Misaka-Mikoto-G** [reply]
thanks
